draft-ietf-httpbis-layered-cookies.md

```diff
@@ -1250,21 +1295,21 @@ boolean _httpOnlyAllowed_, boolean _allowNonHostOnlyCookieForPublicSuffix_, and
 
    Note: The check here and those below are with a byte-lowercased value in order to protect servers that process these values in a case-insensitive manner.
 
-1. If _cookie_'s name, byte-lowercased, starts with `__host-` and not all of the following are true:
+1. If _cookie_'s name, byte-lowercased, starts with `__host-` and _cookie_ is not **Host-prefix compatible**, then return null.
 
-    1. _cookie_'s secure-only is true;
+1. If _cookie_'s name, byte-lowercased, starts with `__http-` and _cookie_ is not **Http-prefix compatible**, then return null.
```

Comment:
Why do we lowercase to `__http-` if it has to be a case-sensitive match with `__Http-` to begin with (above, in prose)?

Reply:
Everyone else were doing it? :shrug:

Should we change the prose?

Reply:
Oh, so there's a difference between the conformance requirements and the processing (which is more lenient). Got it!